src/App.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect, useState } from 'react'
+import axios from 'axios'
+import View from './components/View'
+import Cards from './components/Cards'
```

Comment:
Your components Folder contains tsx and css files for components, usually if react-component wasn't contain only one file, or have multiple sub-components — we create folder to keem them.
```
/components
  /Button
    index.tsx // contains tsx code for Button component
    styles.css // contains all styles for Button component
```